### DISCUSSION TITLE
can I fork the repo to put ASCII art?

### DISCUSSION THREAD

**Discussion by AviAvinav** [lptstr/winfetch#133] [Q&A] [answered]
I really loved winfetch, and thank you for making it.

I wanted to add an ASCII art but it seems it is not possible yet, would it be alright if I fork it and change the art for just my pc(I won't open a pull request)?

**Comment by rashil2000** [comment, marked as answer]
In the current master branch (unreleased), it is possible to input any image and convert it into ASCII art.

Meanwhile, you're free to fork and do whatever you want with the codebase, as long as you comply with winfetch's license, which is MIT: https://github.com/kiedtl/winfetch/blob/master/LICENSE

**Reply by AviAvinav** [reply]
Thanks!
